statsd/end_to_end_udp_test.go

```diff
@@ -229,38 +229,38 @@ func TestTelemetryAllOptions(t *testing.T) {
 
 type testCase struct {
 	opt      []Option
-	testFunc func(*testing.T, *testServer, *Client)
+	testFunc func(*testing.T, *testServer, *ClientEx)
```

Comment:
Wouldn't be better to continue to use `*Client` here? To stay closer to general usage